src/main/java/core/basesyntax/service/handler/Purchase.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.handler;
+
+import java.math.BigDecimal;
+
+public class Purchase implements HandlerByActivity {
```

Comment:
```suggestion
public class PurchaseOperation implements OperationHandler {
```
use same naming for all other handlers